### ISSUE TITLE
Expectations of graduate mentors

### ISSUE THREAD

**Issue by rebeccaradding** [foundersandcoders/master-reference#646]
To provide students with consistent and predictable support during the 16-week course, @sofer has suggested asking that all programme graduates commit to:

1. Mentoring on one of the first 8 weeks of the course 
1. Code reviewing one additional project during weeks 1-8
1. Providing QA on one project during either weeks 10-12 or weeks 13-16 _OR_ mentoring during week 10's design workshops

Signups should happen before the end of the 16 week course.

**Comment by jsms90** [comment]
In London, I've asked @ZooeyMiller & @finnhodgkin to start off the first "mentor preparation" session (see #500) by giving a talk to FAC11 based on the content in this pre-existing issue: #365. Would be happy to get it filmed, to share with other campuses if that would be helpful?

**Comment by sofer** [comment]
@rebeccaradding I don't know that this is a reasonable demand of graduates in Nazareth, where many students are not going to be able to stick around for long unless we can support them in some way. And in Gaza, it's a six-month course.

**Comment by rebeccaradding** [comment]
@sofer **in the case of Nazareth,** I would suggest explaining that this is the minimum level of support that the course requires beyond the part-time facilitator, and that we would like to engage graduates in providing this support, if possible. If not, it will be our responsibility to source it elsewhere. In the case of code reviewing and QA, both can be done remotely, so while the current cohort may not have coverage due to reduced numbers, future cohorts can probably commit to both if aware of the commitment up front.

**In Gaza,** as you say, it's a six-month course. Nevertheless, having a structure like this where people sign up towards the end of the initital 16 weeks to support the following course in specific ways would likely minimize the amount of wrangling the course facilitator would then have to do. Also, the six months do not cover weeks 10-16 of the next course.

**Comment by jsms90** [comment]
This is a pretty hefty time commitment, which essential constitutes an extra week of mentoring:
> Providing QA on one project during either weeks 10-12 or weeks 13-16 OR mentoring during week 10's design workshops

I'm not sure whether it's reasonable to suddenly spring this as an _expectation_ on FAC11 so shortly before their graduation. But I would definitely suggest that we communicate the necessity of this to the cohort and put out a request for volunteers (to mentors and alumni alike) & then add this as an expectation for FAC12+

**Comment by jsms90** [comment]
@matthewdking In general, I think that documentation should be an essential part of a mentor's responsibilities. We can only improve FAC's curriculum together if mentors ensure that whatever material they create/deliver can be be picked up and delivered to the same standard by any future mentor. But I never feel right about expecting this of someone if they haven't been given enough prep time. Especially when it comes to being videoed.

It feels really personal, having a permanent record of you standing up and speaking. So knowing that you're going to be videoed puts on a lot of added pressure. There have been many times in the past when I have asked a mentor if we could film them, so that we could upload this for Naz/Gaza. But that added pressure has almost stopped the mentor from wanting to give the presentation at all.

I'm acutely aware of the fact that @finnhodgkin and @ZooeyMiller are both currently giving back to FAC with a lot of their free time as Week Facilitators & have each committed far more volunteering time as mentors than FAC expects from anyone. In fact, that's exactly why I've asked them to step in for this bit. FAC11 now trust and respect them a great deal, which is why it is perfect for them to be outlining FAC's expectations for mentorship.

I really haven't given @finnhodgkin or @ZooeyMiller much advanced warning about this idea at all. So I wouldn't want them to feel pressured into being videoed. But I'm very committed to the idea of making any curriculum content deliverable across campuses! I will certainly take notes, in case anything is covered other than the content in the top comment & post those in here for you guys to use. Please let me know if there's anything else you can think of that would be helpful?

**Comment by sofer** [comment]
@jsms90 I agree that springing anything on the existing cohort is not a reasonable expectation. I am keen to set expectations for the new cohorts starting in a few weeks.

**Comment by mantagen** [comment]
@sofer you mentioned to me the idea of groups of 4 mentoring across 2 weeks of the course.
Or was I imagining it?

**Comment by ZooeyMiller** [comment]
Here are the slides we used. We spoke in words some as well to clarify points and tlakbout other things as well though. https://docs.google.com/presentation/d/1VG_-Pm5byZyzWrmZ0dH7r-0pUON1jxc_8W0OMn2sm5I/edit?usp=sharing

**Comment by mantagen** [comment]
@ZooeyMiller Talk looks very good, gonna copy and adapt for Gaza audience.
Many thanks
